src/lib/bootloader/exceptions.rb

```diff
@@ -26,6 +26,27 @@ def initialize(msg)
     end
   end
 
+  # Specialized exception for invalid by-path device names
+  # (bsc#1122008, bsc#1116305)
+  class BrokenByPathDeviceName < RuntimeError
+    include Yast::I18n
+    attr_reader :dev_name
+
+    def initialize(dev_name)
+      @dev_name = dev_name
+      textdomain "bootloader"
+
+      # TRANSLATORS: %s is the device name
+      super _("Error reading the bootloader configuration files:\n" \
```

Comment:
NP: I am fine with `%` usage for formatting the string, but it would be preferable to use `format` method, that is:

`format(_("... %{dev} ..."), dev: dev_name)`

Reply:
In fact, I believe that `%` and `format` are basically aliases. So you can use both specifications (`%{var_name}` or `%s`) with both methods. And in general I also prefer the `%{variable_name}` way.

BUT translators are more likely to mess up things when we use the Ruby-specific way than when we use the more standard `%s` one. So I would stick if possible to the more widespread format.